Backend/app/dashboard/Lib/src/Domain/Types/Transaction.hs

```diff
@@ -77,5 +78,6 @@ data Endpoint
   | MapsAPI BAP.MapEndPoints
   | FlowStatusAPI BAP.RideNotifyEventEndPoint
   | CancelAPI BAP.RideCancelEndPoint
+  | TripRouteAPI BAP.RouteEndPoint
```

Comment:
Transaction type is not required for Get API, it is mostly for POST APIs

Reply:
@hkmangla  by the way somehow this is POST api:

```
type TripRouteAPI =
  "trip"
    :> "route"
    :> Capture "rideId" (Id DP.Ride)
    :> ReqBody '[JSON] TripRouteReq
    :> Post '[JSON] Maps.GetRoutesResp
```